src/app/home/page.tsx

```diff
@@ -0,0 +1,169 @@
+        alignItems: "center",
+      }}
+    >
+      <Box
+        sx={{
+          width: "100%",
+          display: "flex",
+          justifyContent: "space-between",
+          alignItems: "center",
+        }}
+      >
+        <Typography
+          variant="h6"
+          sx={{ fontWeight: 600, color: "#333", fontFamily: "Outfit, sans-serif" }}
+        >
+          {name}
+        </Typography>
+        <Typography
+          variant="body2"
+          sx={{
+            color: "#6C6C80",
+            fontSize: "16px",
+            lineHeight: "20.16px",
+            fontFamily: "Outfit, sans-serif",
+          }}
+        >
+          {date}
+        </Typography>
+      </Box>
+
+      <Box
+        sx={{
+          width: "400px",
+          height: "400px",
+          backgroundColor: "#F0F0F5",
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          borderRadius: "8px",
+        }}
+      >
+        <Typography
+          sx={{
+            color: "#6C6C80",
+            fontSize: "16px",
+            fontWeight: 400,
+            fontFamily: "Outfit, sans-serif",
+          }}
+        >
+          Name or Logo or Demo
+        </Typography>
+      </Box>
+
+      <Box
+        sx={{
+          width: "100%",
+          display: "flex",
+          justifyContent: "space-around",
+          alignItems: "center",
+        }}
+      >
+        <Box sx={{ textAlign: "center", display: "flex", alignItems: "center", gap: "8px" }}>
+          <VisibilityOutlinedIcon sx={{ color: "#000000", fontSize: "36px" }} />
+          <Typography sx={{ fontSize: "14px", color: "#333", fontFamily: "Outfit, sans-serif" }}>
+            {views}
+          </Typography>
+        </Box>
+        <Box sx={{ width: "1px", backgroundColor: "#D6D6E7", height: "48px" }} />
+
+        <Box sx={{ textAlign: "center", display: "flex", alignItems: "center", gap: "8px" }}>
+          <ModeCommentOutlinedIcon sx={{ color: "#000000", fontSize: "36px" }} />
+          <Typography sx={{ fontSize: "14px", color: "#333", fontFamily: "Outfit, sans-serif" }}>
+            {comments}
+          </Typography>
+        </Box>
+        <Box sx={{ width: "1px", backgroundColor: "#D6D6E7", height: "48px" }} />
+
+        <Box sx={{ textAlign: "center", display: "flex", alignItems: "center", gap: "8px" }}>
+          <FavoriteBorderOutlinedIcon sx={{ color: "#000000", fontSize: "36px" }} />
+          <Typography sx={{ fontSize: "14px", color: "#333", fontFamily: "Outfit, sans-serif" }}>
+            {likes}
+          </Typography>
+        </Box>
+      </Box>
+
+      <Button
+        variant="contained"
+        className="custom-next-button"
+        sx={{
+          width: "fit-content",
+          minWidth: 120,
+          padding: "8px 16px",
+          whiteSpace: "nowrap",
+          flexShrink: 0,
+          fontFamily: "Outfit, sans-serif",
+        }}
+      >
+        Explore More
+      </Button>
+    </Box>
```

Comment:
I really like the final result and I think you have done a solid job in replicating the design in the figma document!

Reply:
Finally please run npm run format to pass the formatting checks in our CI/CD pipeline.

Reply:
Done